superset-frontend/src/cccs-viz/plugins/plugin-chart-cccs-grid/src/CccsGrid.tsx

```diff
@@ -209,6 +209,15 @@ export default function CccsGrid({
     return formData.column_config?.[col]?.emitTarget || col;
   }
 
+  function autoSizeFirstColumns(params: any){
```

Comment:
Tempted to name this autosizeFirst100Columns. Or at least add a comment to say we're autosizing the first 100.